CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 Changelog
 =========
 
-[v2.19](https://github.com/rigetti/pyquil/compare/v2.18.0...master) (in development)
+[v2.19](https://github.com/rigetti/pyquil/compare/v2.18.0...master) (March 26, 2020)
```

Comment:
Add I guess this should be `v2.18.0...v2.19.0`?

Reply:
IIRC, I made this exact comment to piotr once and he said he usually waits until after the release when the tag actually exists.

Reply:
Throw off the shackles of Peters past. Be the change you seek.

Reply:
I'm sure it probably doesn't really matter, but for example here are the two commits from the last karalekas release:

[version bump for 2.17.0](https://github.com/rigetti/pyquil/commit/2402cae0b72ff6232b81ec8899bd5b741a94916d)
[follow on commit to update changelog for 2.18.0](https://github.com/rigetti/pyquil/commit/e916a4d38e26910e73d70f6778e7657f0a292b52)

Reply:
Seems perfectly reasonable to do it here but 🤷‍♀

Reply:
You kids today have no respect for the traditions of the past (smh).